src/BloomExe/Workspace/WorkspaceView.cs

```diff
@@ -168,6 +169,32 @@ LocalizationManager localizationManager
 			SetupUILanguageMenu();
 		}
 
+		/// <summary>
+		/// Adjusts the tool panel location to allow more (or optionally less) space
+		/// for the tab buttons.
+		/// </summary>
+		void AdjustToolPanelLocation(bool allowNarrowing)
+		{
+			var x = _tabStrip.Items.Cast<TabStripButton>().Sum(tab => tab.Width) + 10;
```

Comment:
So the idea here is that the toolSpecificPanel should be placed just right of the space required for all the tabs? Initially we leave just enough room for the current localization, but as localization changes we don't move them back?
It might be helpful to give x an informative name, like widthOfTabsLeftOfToolPanel.